api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -418,12 +417,12 @@ public void getCohortsContainingPatient_shouldNotReturnVoidedCohorts() throws Ex
 		assertFalse(service.getCohort(2).isVoided());
 		
 		// add a patient to both cohorts
-		Patient patientToAdd = new Patient(4);
+		Patient patientToAdd = new Patient(7);
```

Comment:
I think I get it now, before the member was just an integer and now it's an entity that has a reference, so you had to change to a patient with an Id that exists in the DB, that makes sense.